scripts/proxy.py

```diff
@@ -4,54 +4,79 @@
 import sys
 import inspect
 import traceback
+import sj
+import typing
 
-from distutils.version import LooseVersion
+from mitmproxy import http
+from mitmproxy import ctx
+from mitmproxy.script import concurrent
 
 from subprocess import CalledProcessError, Popen, PIPE, STDOUT
 
-p = Popen(['mitmdump --version'], stdout=PIPE, stdin=PIPE, stderr=STDOUT, shell=True)
+'''p = Popen(['mitmdump --version'], stdout=PIPE, stdin=PIPE, stderr=STDOUT, shell=True)
```

Comment:
Why are these code lines being turned into string constants?  Can we just delete them?